packages/dlp/test/gapic_v2beta1.js

```diff
@@ -0,0 +1,341 @@
+/*
```

Comment:
I agree that hyphen is slightly preferable (for consistency), but either one is fine.
If I used an underscore in video intelligence, it is totally in bounds to rename it. :-)